src/components/GettingStartedModal/ChooseSetStep.tsx

```diff
@@ -0,0 +1,111 @@
+import { ChangeEvent, FormEvent, ReactElement, useEffect , useState} from "react";
+import { NavigationButtons, Step, StepProps } from ".";
+import { Button } from "../Button";
+import {
+  isPhoneticsPreference,
+  PhoneticsPreference,
+  PREFERENCE_LITERATES,
+} from "../../state/reducers/phoneticsPreference";
+import { GROUPS, isGroupId } from "../../state/reducers/groupId";
+import { SectionHeading } from "../SectionHeading";
+import { CollectionDetails } from "../CollectionDetails";
+
+import { collections, VocabSet } from "../../data/vocabSets"; 
+import { useUserStateContext } from "../../providers/UserStateProvider";
+import { TermsByProficiencyLevelChart } from "../TermsByProficiencyLevelChart";
+
+export const ChooseSetStep: Step = {  
+  title: "Choose your first set",
+  commitState: ({ collectionId }, {setUpstreamCollection}) => {
+    if (collectionId !== undefined){
+      setUpstreamCollection(collectionId);
+    }
+  },
+  Component: ChooseStepComponent,
+};
+
+function ChooseStepComponent({
+  wizardState: { collectionId, groupId },
+  setWizardState,
+  goToNextStep,
+  goToPreviousStep,
+}: StepProps): ReactElement {
+  var canGoToNextStep = collectionId !== undefined; 
```

Comment:
I think this should be `const` -- `var` is evil and has terrible behavior!

Reply:
I changed it to let so we'll be able to reassign it later.